include/aspect/mesh_deformation/ascii_data.h

```diff
@@ -0,0 +1,99 @@
+/*
+  Copyright (C) 2016 - 2019 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef _aspect_mesh_deformation_model_ascii_data_h
+#define _aspect_mesh_deformation_model_ascii_data_h
```

Comment:
As a convention we use the directory/file structure for the include guards, so your code works, but please change to be consistent with the rest of the files.

```suggestion
#ifndef _aspect_mesh_deformation_ascii_data_h
#define _aspect_mesh_deformation_ascii_data_h
```